core/batch_loader.py

```diff
@@ -286,7 +286,7 @@ def _load_issue(self, mets_file):
                 page = self._load_page(doc, page_div, issue)
                 self.pages_processed += 1
             except BatchLoaderException, e:
-                _logger.exception(e)
+                LOGGER.exception(e)
```

Comment:
Think it's worth fixing this while you're in here? logger.exception() is supposed to be called with a msg & other formatting arguments — it'll automatically pull the exception in, so this ends up being somewhat redundant. I usually like to have the message indicate some context which might not be in the exception message – e.g. something like `LOGGER.exception("Unable to load issue %s page %s", issue, page_div)` except that you might need something else to make that `page_div` output useful.

Also, while we're in here I'm really tempted to modernize all of the `except` blocks to the non-deprecated `except … as` syntax.

Reply:
I think it is worth fixing, but I think we should do those 2 things in a new pull request